### PR TITLE
wifi : Bugfix for RRM, BTM request handling

### PR DESCRIPTION
**Pull request by sr1dh48r**
[SHEL-877]: DUT did not respond to AP beacon request frame
Enable RRM request/response frame handling.

[SHEL-859]: No BTM response for request from AP
Handle deauth during BSS transition

Signed-off-by: Sridhar Nuvusetty <sridhar.nuvusetty@nordicsemi.no>

### CODE REVIEW

File: src/drivers/driver_zephyr.c [hunk 1]
```diff
@@ -1121,7 +1129,6 @@ static int wpa_drv_zep_set_key(void* priv,
 static int wpa_drv_zep_get_capa(void *priv,
 			       	struct wpa_driver_capa *capa)
 {
-
```
Comment (krish2718) [inline src/drivers/driver_zephyr.c:1124]: unnecessary whitespace

File: src/utils/os_zephyr.c [hunk 2]
```diff
@@ -45,6 +45,8 @@ int os_get_reltime(struct os_reltime *t)
 	struct timespec ts;
 	int res;
 
+	clock_id = CLOCK_REALTIME;
```
Comment (krish2718) [inline src/utils/os_zephyr.c:48]: This should be a separate commit with appropriate commit log

Comment (krish2718) [reply src/utils/os_zephyr.c:48]: FYI, I have root cause and raised https://github.com/nrfconnect/sdk-hostap/pull/53 with proper fix (added your SOB too). So, no need to switch to REALTIME, we can get it working with any clock setting.

**Review by sachinthegreen — APPROVED** [review]
